tests/unit/test_text_splitters.py

```diff
@@ -0,0 +1,54 @@
+#  Neo4j Sweden AB [https://neo4j.com]
+#  #
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  #
+#      https://www.apache.org/licenses/LICENSE-2.0
+#  #
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+import pytest
+from langchain_text_splitters import RecursiveCharacterTextSplitter
+from llama_index.core.node_parser.text.sentence import SentenceSplitter
+from neo4j_genai.generation.text_splitters import TextSplitterAdapter
+
+pytest_plugins = ("pytest_asyncio",)
```

Comment:
Side comment: does it fail if you remove this line? I'm asking because I didn't (even though I also installed this plugin and have async tests), and I had the impression tests were running, but maybe I'm wrong

Reply:
Have just tested it and it works also when I remove it. Shall I remove it from the PR?

Reply:
If it has no effect, I'd say we can remove. Are you afraid of any side effect or something?

Reply:
Done!